### DISCUSSION TITLE
Can't verify my Github account

### DISCUSSION THREAD

**Discussion by thaddeusgamit** [community/community#49487] [GitHub Education]
I'm always getting this email when I'm verifying my self for internship  

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Mar 8, 2023 for [@Teejhayzxc](https://github.com/Teejhayzxc).

You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio. 

although I already change my profile and I'm submitting Certification of Registration

**Comment by Peins** [comment]
Me too

**Comment by Gusarovv** [comment]
I found the following solution:
The problem is that this "error" also includes the fact that you are uploading an identity in a non-English language. Try to translate the ID and re-upload.

**Reply by Peins** [reply]
> I found the following solution: The problem is that this "error" also includes the fact that you are uploading an identity in a non-English language. Try to translate the ID and re-upload.

My all prove files are all English,but it shows this too.Besides,I already change my profile.I tried many times but it dosen't work.

**Reply by aviiciii** [reply]
Yes, same problem for me!

**Reply by k0mr4d3** [reply]
Same problem for me!

**Reply by wenjuan7275** [reply]
I logged out of the account, then logged in again, and the review was passed.

**Comment by ranpin** [comment]
me too,i have tried more than 20 times.

**Comment by BiscuitFingers** [comment]
I'm also having the same issue. I have tried a few different ID methods, still the same issue
